xds/src/main/java/io/grpc/xds/XdsNameResolverProvider.java

```diff
@@ -35,6 +36,7 @@
  * through {@link io.grpc.NameResolverRegistry#asFactory#newNameResolver(URI, Args)} with the URI
```

Comment:
Can you remove this paragraph? This tag is causing javadoc warning.

Reply:
Fixed.